reV/qa_qc/summary.py

```diff
@@ -207,7 +207,9 @@ def summarize_means(self, out_path=None):
         with Resource(self.h5_file, group=self._group) as f:
             meta = f.meta
             if 'gid' not in meta:
-                meta.index.name = 'gid'
+                if meta.index.name == 'gid':
+                    meta.index.name = 'gid'
```

Comment:
seems redundant?

Reply:
whoops, good call, was trying to Not run that unless we need to